assets/css/index.css

```diff
@@ -0,0 +1,13 @@
+.list-group {

```

Comment:
Visa karkaso ir `git` idėja ir yra **nedublikuoti** savo kodo.
Jei gerai supratau tau PHPStorm (ar pan.) sugeneruoja iš `index.sass` -> `index.css` failą.
Bet realybėje jo nebūtina dėti į Versijų kontrolę (`GitHub`), nes `index.sass` -include-> `app.css` -konfiguracija-> `webpack.config.js` -> `public/build/app.css`
Taigi `index.css` ir `build/app.css` dublikuoja kodą.

Taigi,
* Per PHPStorm (ar pan.) išjunk Watcher's/auto compile. Nes `yarn run encore dev --watch` daro lygiai tą patį
* Arba bent jau įdėk šį failą į `.gitignore`
* (kadangi jau sukėliai į GitHub) reikėtų (išmokti) jį ištrinti. Žr, https://stackoverflow.com/questions/7527982/applying-gitignore-to-committed-files